src/Hamiltonians/HubbardRealSpace.jl

```diff
@@ -228,9 +289,24 @@ function Base.show(io::IO, h::HubbardRealSpace)
     println(io, ")")
 end
 
+"""
+    ==(H::HubbardRealSpace, G::HubbardRealSpace)
+
+Overload equality due to stored potential energy arrays.
+"""
```

Comment:
```suggestion
# Overload equality due to stored potential energy arrays.
```
I would skip the docstring here. This will appear if you use `?==` in the repl.